### ISSUE TITLE
[wrangler tail logs] Introduce installer logic for cloudflared

### ISSUE THREAD

**Issue by gabbifish** [cloudflare/wrangler#924]
We need to ensure that wrangler can download the cloudflared binary if necessary; this will require adding a new set of download logic specific to equinox.io, which is where we host our cloudflared binaries.

**Comment by ashleymichal** [comment]
this is blocked on cloudflared publishing binaries on a host with a stable API.

**Comment by Electroid** [comment]
Tunnel team now publishes releases to Github: https://github.com/cloudflare/cloudflared/releases

**Comment by EverlastingBugstopper** [comment]
still no macos binaries unfortunately

**Comment by Electroid** [comment]
Won't fix: moving away from `cloudflared` and using WebSockets instead.
